### PR TITLE
Improve logs for SEP10Service

### PR DESCRIPTION
**Pull request by Ifropc**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Forward changes of #1023 

### Why

Better logs

### Known limitations

N/A

### CODE REVIEW

File: core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java [hunk 1]
```diff
@@ -224,14 +224,17 @@ public String validateChallenge(String challengeXdr)
...
+          "Verifying challenge threshold. server_account={}, client_domain={}, signers={}",
           shorter(serverAccountId),
+          clientDomain,
           signers.size());
 
       if ((clientDomain != null && challenge.getTransaction().getSignatures().size() != 3)
           || (clientDomain == null && challenge.getTransaction().getSignatures().size() != 2)) {
         infoF(
-            "Invalid SEP 10 challenge exception, there is more than one client signer on challenge transaction for an account that doesn't exist");
+            "ALERT: Invalid SEP 10 challenge exception, there is more than one client signer on challenge transaction for an account that doesn't exist. client_domain={}, account_id={}",
```
Comment (lijamie98) [inline core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java:235]: `ALERT:` is MGI specific and lack of consistency. We should avoid anchor-specific implementation.

**Review by lijamie98 — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java (hunk 1 above)
Comment (Ifropc) [reply core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java:235]: Yep sorry was just a copy-paste

**Review by philipliu — APPROVED** [review]
LGTM

**Review by lijamie98 — APPROVED** [review]
LGTM

**Pull request merged**